docs/content/ingestion/stream-pull.md

```diff
@@ -166,6 +172,13 @@ The following policies are available:
 * `messageTime` &ndash; Can be used for non-"current time" as long as that data is relatively in sequence. Events are rejected if they are less than `windowPeriod` from the event with the latest timestamp. Hand off only occurs if an event is seen after the segmentGranularity and `windowPeriod` (hand off will not periodically occur unless you have a constant stream of data).
 * `none` &ndash; All events are accepted. Never hands off data unless shutdown() is called on the configured firehose.
 
+### Index Spec
+
+|Field|Type|Description|Required|
+|-----|----|-----------|--------|
+|bitmap|string|type of bitmap to use (e.g. roaring or concise), null to use the default.Defaults to the bitmap type specified by the (deprecated) "druid.processing.bitmap.type" setting| No|
```

Comment:
defaults to concise in `io.druid.segment.IndexSpec#IndexSpec(io.druid.segment.data.BitmapSerdeFactory, java.lang.String, java.lang.String)`